nym-vpn-core/crates/nym-vpn-lib/src/platform/mod.rs

```diff
@@ -257,12 +257,12 @@ async fn fetch_account_links(
     network_name: &str,
     locale: &str,
 ) -> Result<AccountLinks, VpnError> {
-    let account_id = account::get_account_id(path).await?;
+    let account_id = account::get_account_id(path).await.ok();
```

Comment:
same comment as in `nym-vpn-core/crates/nym-vpnd/src/service/vpn_service.rs` regarding possible error matching